netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+  publish = "docs/_build/html"
+  command = "pip install -r docs-requirements.txt && export PYTHONPATH=$(pwd) && python setup.py docs"
```

Comment:
I may be wrong, but shouldn't we not be having the need to export ```PYTHONPATH``` if sphinx has its own conf file and the repository is always relative to it ?

Reply:
Spotted something,
We do [set the path in conf.py](https://github.com/coala/coala/blob/master/docs/conf.py#L25)
but since we do the [import before setting the path]((https://github.com/coala/coala/blob/master/docs/conf.py#L20)), the path being set is actually useless

Reply:
Oh
So does setting that path serve any other purpose then ?
If not then should we remove that as well (possible the removal would be a non-standard inclusion, but we get rid of some un-used code maybe) ?

Reply:
I suspect that the only reasons for it was so that "setup.py docs" would work.

It used to work.  See 687ec1bc8 which broke it.  And https://github.com/coala/coala/issues/3785 for the original issue.

Probably broken atm.  I've raised https://gitlab.com/coala/mobans/issues/8

Until we have a verified solution again, that line should stay in the local docs/conf.py .  Worst case it will disappear when we mobanise conf.py.

Reply:
Well, turns out that there is a issue which should be resolved in mobans.  See https://gitlab.com/coala/mobans/merge_requests/11